client-failover/client-failover-sample-hazelcast5/src/main/java/sample/com/hazelcast/clientfailover/ClientWithSsl.java

```diff
@@ -0,0 +1,127 @@
+        props.setProperty("javax.net.ssl.trustStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        ClientConfig config = new ClientConfig();
+        config.getNetworkConfig().setRedoOperation(true);
+        config.getNetworkConfig().setSSLConfig(new SSLConfig().setEnabled(true).setProperties(props));
+        config.getNetworkConfig().getCloudConfig()
+            .setDiscoveryToken("YOUR_DISCOVERY_TOKEN")
+            .setEnabled(true);
+        config.setProperty("hazelcast.client.cloud.url", "YOUR_CLIENT_URL");
+        config.setClusterName("YOUR_CLUSTER_ID");
+        config.getConnectionStrategyConfig()
+            .getConnectionRetryConfig()
+            .setClusterConnectTimeoutMillis(10000); // <5>
+        return config;
+        }
+    
+    // <6>
+    private static ClientConfig getSecondaryClientConfig() throws Exception {
+        ClassLoader classLoader = ClientWithSsl.class.getClassLoader();
+       Properties props = new Properties();
+        props.setProperty("javax.net.ssl.keyStore", classLoader.getResource("client2.keystore").toURI().getPath()); // <7>
+        props.setProperty("javax.net.ssl.keyStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        props.setProperty("javax.net.ssl.trustStore",
+            classLoader.getResource("client2.truststore").toURI().getPath()); // <7>
+        props.setProperty("javax.net.ssl.trustStorePassword", "YOUR_KEYSTORE_PASSWORD");
+        ClientConfig config = new ClientConfig();
+        config.getNetworkConfig().setRedoOperation(true);
+        config.getNetworkConfig().setSSLConfig(new SSLConfig().setEnabled(true).setProperties(props));
+        config.getNetworkConfig().getCloudConfig()
+           .setDiscoveryToken("YOUR_DISCOVERY_TOKEN")
+           .setEnabled(true);
+        config.setProperty("hazelcast.client.cloud.url", "YOUR_CLIENT_URL");
+        config.setClusterName("YOUR_CLUSTER_ID");
+        config.getConnectionStrategyConfig().getConnectionRetryConfig().setClusterConnectTimeoutMillis(10000);
+        return config;
+        }
+
+    /**
+     * This example shows how to work with Hazelcast maps, where the map is
+     * updated continuously.
+     *
+     * @param client - a {@link HazelcastInstance} client.
+     */
+    private static void nonStopMapExample(HazelcastInstance client) {
+        System.out.println("Now the map named 'map' will be filled with random entries.");
+
+        IMap<String, String> map = client.getMap("map");
+        Random random = new Random();
+        int iterationCounter = 0;
+        while (true) {
+            int randomKey = random.nextInt(100_000);
+            // tag::catch-exceptions[]
+            try {
+                map.put("key-" + randomKey, "value-" + randomKey);
+            } catch (Exception e) {
+                // Captures exception from disconnected client
+                System.out.println("Exception (" + e.getClass().getCanonicalName() + ") " + e.getMessage());
```

Comment:
minor: Maybe we can use `e.printStackTrace()` instead?

Reply:
Updated as requested.